internal/test/scorecard/tests.go

```diff
@@ -288,3 +288,89 @@ func logEvents(r *scapiv1alpha3.TestResult, client *CryostatClientset, namespace
 	}
 	return nil
 }
+
+func newCryostatCR(namespace string, withIngress bool) *operatorv1beta1.Cryostat {
```

Comment:
Could you split out these changes to a separate PR? I think they make sense to separate out in the commit history.

Reply:
yes just did: https://github.com/cryostatio/cryostat-operator/pull/589